src/ui.rs

```diff
@@ -85,7 +85,13 @@ fn draw_header<B: Backend>(
         .insert(UiTarget::StockName, stock_name_area);
 }
 
-fn draw_body<B: Backend>(f: &mut Frame<B>, App { stock, .. }: &App, area: Rect) {
+fn draw_body<B: Backend>(
+    f: &mut Frame<B>,
+    App {
+        stock, ui_state, ..
```

Comment:
Why do we need a `..` here?

Reply:
`rustc` bug? Haha... Are you using nightly? I'm on stable...

Reply:
Previously you also have the `..`, just curious is it needed?

Reply:
Previously I wasn't using `ui_state` here... So no, it's not needed anymore if you use everything.